tests/ansible_collections/basic_setup.yml

```diff
@@ -3,6 +3,8 @@
     # to be overridden by the tmt
     repo_root:
   tasks:
+    # including rhel internal repos for tests purposes
+    - include: copy_internal_rhel_repos.yml
```

Comment:
this is very cool. I had some problems in the past to include other playbooks. This means we can use Ansible roles with vars